src/server_manager/web_app/shadowbox_server.ts

```diff
@@ -54,8 +54,15 @@ function makeAccessKeyModel(apiAccessKey: AccessKeyJson): server.AccessKey {
   return apiAccessKey as server.AccessKey;
 }
 
+// Represents either a JSON payload or a FormData payload.
+interface ApiRequestBody {
```

Comment:
I'd get rid of this and just require the body to be a string.

Reply:
Done